frontend/src/components/Output/ModelParameters.js

```diff
@@ -1,10 +1,7 @@
-import _ from "lodash";
 import React, {Component} from "react";
 import {observer} from "mobx-react";
 import PropTypes from "prop-types";
-
```

Comment:
this is a python thing that i'm applying to js, but i like having 3rd party packages imported with a linebreak between my custom code packages. makes it easy to better understand at a glance what comes from where

Reply:
imports separated.